xtask/src/check_feature_dependencies.rs

```diff
@@ -0,0 +1,114 @@
+use pico_args::Arguments;
+use xshell::Shell;
+
+#[derive(Debug)]
+#[expect(dead_code)]
```

Comment:
nitpick: lolwut, `reason` for `Positive` plz?

question: Also, is it not possible to narrow this down to that specific variant? 🤔

Reply:
`expect` (now that it's finally available) is pretty much always better than `allow` - the problem with `allow` is that it won't go away in the future as you can't catch the circumstances changing

.. if that's what you mean by `Positive`

Reply:
I thought I was going to write a positive check, but I didn't immediately think of one. I'm going to leave it for now, but will move the expect.